src/Controllers/BatchProcessorController.js

```diff
@@ -308,21 +311,66 @@ var StreamStats;
                 }
                 return;
             };
-            BatchProcessorController.prototype.submitBatch = function () {
-                var url = null;
+            BatchProcessorController.prototype.submitBatch = function (submit250) {
+                var _this = this;
+                if (submit250 === void 0) { submit250 = false; }
+                this.addStatIDtoList();
                 var formdata = new FormData();
-                formdata.append('submit_batch[email]', this.submitBatchData.email);
-                formdata.append('submit_batch[subject]', this.submitBatchData.idField);
-                formdata.append('submit_batch[attachments][][resource]', this.submitBatchData.attachment, this.submitBatchData.attachment.name);
-                this.toaster.pop('success', "The batch was submitted successfully. You will be notified by email when results are available.", "", 5000);
+                formdata.append('region', this.selectedRegion.toString());
+                formdata.append('basinCharacteristics', this.selectedParamList.toString());
+                formdata.append('flowStatistics', this.flowStatIDs.toString());
+                formdata.append('email', this.submitBatchData.email.toString());
+                formdata.append('IDField', this.submitBatchData.idField.toString());
+                formdata.append('geometryFile', this.submitBatchData.attachment, this.submitBatchData.attachment.name);
                 var headers = {
-                    "tdb": "tbd"
+                    "Content-Type": undefined
                 };
-                var request = new WiM.Services.Helpers.RequestInfo(url, true, WiM.Services.Helpers.methodType.POST, 'json', formdata, headers, angular.identity);
                 this.submittingBatch = true;
+                if (submit250 == true) {
+                    formdata.append('moreThan250Points', submit250.toString());
+                    this.postBatchFormData(formdata, headers).then(function (response) {
+                        var r = response;
+                        if (r.status == 200) {
+                            _this.toaster.pop('success', "The batch was submitted successfully. You will be notified by email when results are available.", "", 20000);
+                        }
+                        else {
+                            var detail = r.data.detail;
+                            _this.toaster.pop('error', "The submission failed for the following reason:" + detail, "", 5000);
```

Comment:
![image](https://github.com/USGS-WiM/StreamStats/assets/40237491/ac175b98-feef-4876-ac9b-3317d71dcd82)

Add space after the colon

```suggestion
                            _this.toaster.pop('error', "The submission failed for the following reason: " + detail, "", 5000);
```